apps/manifest.go

```diff
@@ -187,8 +186,7 @@ func (m Manifest) IsValid() error {
 		return utils.NewInvalidError(errors.New("homepage_url is empty"))
 	}
 
-	_, err := url.Parse(m.HomepageURL)
-	if err != nil {
+	if err := utils.IsValidHTTPURL(m.HomepageURL); err != nil {
```

Comment:
This function is available from `mattermost-server/model`. Do we want to import it from there?

Reply:
I don't think so for two reasons:

1. It's not worth including the full package from `model` for a single methods. This follows "A little copying is better than a little dependency.".
2. The `model` method doesn't return an  `error`, which I don't like, because we can't return a meaningful on to the user then.